SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -210,6 +210,7 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 													  const LocalDeploymentCallback& CallBack)
 {
 	RuntimeStartTime = FDateTime::Now();
+	bRedeployRequired = false;
```

Comment:
Driveby where the "Redeploy" flag was never set to false after being enabled. This resulted in deployments restarting even when not desired. @joshuahuburn